pgzx-update-0.2.0.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: 'pgzx: Version 0.2.0 update'
+description: 'Learn about the latest changes in pgzx'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgzx-banner.png
+  alt: pgzx
+author: Steffen Siering
+authorEmail: steffen@xata.io
+date: 07-19-2024
+tags: ['open-source', 'postgres', 'fpPgzx', 'oss']
+published: false
+slug: pgzx-update-0.2.0
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgzx-banner.png
+---
+
+We introduced [pgzx](https://github.com/xataio/pgzx) during our last launch
```

Comment:
```suggestion
We introduced [pgzx](https://xata.io/pgzx) during our last launch
```

Reply:
Adding our feature page here as good for SEO. Will mention github a little further down.